install/helm/agones/values.yaml

```
@@ -92,6 +92,15 @@ agones:
     replicas: 2
     pdb:
         minAvailable: 1
+    topologySpreadConstraints:
+      enabled: true
+      constraints:
+        - maxSkew: 1
+          topologyKey: kubernetes.io/hostname
+          whenUnsatisfiable: ScheduleAnyway
+          labelSelector:
+            matchLabels:
+              app: agones-controller
```

```suggestion
    topologySpreadConstraints: {}
```

(And same for all the other ones)

Since we don't want to set a default value -- but this was a good way to test the indent level, since we can see it in the index.yaml.